src/components/WelcomeModal/CarouselCard.tsx

```diff
@@ -0,0 +1,75 @@
+import { useRef, useEffect } from "react";
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import { Body, Subtitle } from "@leafygreen-ui/typography";
+import { size } from "constants/tokens";
+import { CardType } from "./types";
+
+interface CarouselCardProps {
+  card: CardType;
+  visible: boolean;
+}
+
+const CarouselCard: React.VFC<CarouselCardProps> = ({ card, visible }) => {
+  const { img, video, subtitle, title, description, alt } = card;
+  const videoRef = useRef<HTMLVideoElement>(null);
+  useEffect(() => {
+    if (visible && videoRef.current) {
+      const playPromise = videoRef.current.play();
+      // https://developer.chrome.com/blog/play-request-was-interrupted/
+      if (playPromise !== undefined) {
+        playPromise.catch((err) => {
+          if (err.name === "NotAllowedError") {
+            // User blocked autoplay.
+          }
+        });
+      }
+    }
+  }, [visible]);
+  return (
+    <CardContainer>
```

Comment:
Bump on disabling text highlight? You could wrap the text in a `TextContainer` and have the styles be `cursor: default; user-select: none;` 👀